getting-started/docker.md

```diff
@@ -0,0 +1,134 @@
+   Coder-provided images hosted in a Docker registry). Select **Ubuntu**. This
+   will populate the form in the **Import** tab.
+
+1. Under **Workspace providers**, leave the default option (which is **Docker**)
+   selected.
+
+1. Scroll to the bottom, and click **Create workspace**. The dialog will close,
+   allowing you to see the main workspace page. You can track the workspace
+   build process using the **Build log** on the right-hand side.
+
+![Create a workspace](../assets/getting-started/create-workspace.png)
+
+Once your workspace is ready for use, you'll see a chip that says **Running**
+next to the name of your workspace.
+
+## Step 3: Begin using Coder
+
+Once you've created your workspace, you can start using Coder.
+
+1. Under **Browser applications**, click **Code Web** to open VS Code in your
+   browser.
+
+1. When VS Code launches in your browser, click **Open folder...**. In the
+   prompt, you'll see `/home/coder`. This directory is where you'll clone a
+   sample React app project Git repository. Click **OK** to proceed.
+
+   ![Open folder](../assets/getting-started/open-folder.png)
+
+1. Click the hamburger icon in the top right, and select **Terminal** > **New
+   Terminal** to open a new terminal.
+
+1. Clone the `create-react-app` repository:
+
+   ```console
+   git clone https://github.com/facebook/create-react-app.git
+   ```
+
+   Once you've cloned the repository, you can click the **Explorer** icon and
+   expand the **create-react-app** folder in the left-have nav bar to see the
+   contents of the repo.
+
+   ![View files](../assets/getting-started/view-files.png)
+
+   You're now ready to make changes to the application.
+
+## Step 4: Preview your app and view changes live
+
+Dev URLs allow you to access the web services you're developing in your
+workspace. Once you've created a dev URL, Coder listens on the port you
+specified and renders a browser link you can use to view your application.
+
+1. Return to your workspace overview page, and find the **Dev URLs** section.
+
+1. Click **Add port**.
+
+1. Provide a **Name** for your port, and leave the remaining fields as-is. Click
+   **Save**.
+
+   ![Create dev URL](../assets/getting-started/create-devurl.png)
+
+1. At this point, you can build and run the sample app by returning to your Code
+   Web window and running the following in the terminal:
+
+   ```console
+    npx create-react-app my-app
+    cd my-app
+    npm start
+   ```
+
+1. From the workspace overview, launch your dev URL by clicking its name; Coder
+   will open a new browser window and point you to the appropriate URL.
+
+   ![Launch dev URL](../assets/getting-started/launch-devurl.png)
+
+1. You can test preview by making changes to the `src/App.js` file; every time
+   you save your changes to this file and refresh your browser window, your
+   preview will update.
+
+   ![Preview changes](../assets/getting-started/hello-world.png)
```

Comment:
I see we don't include this image in the "developer" getting started guide. Any reason for this?

Reply:
I forgot. Thanks for calling this out!